docs/index.rst

```diff
@@ -447,7 +447,7 @@ Footnotes
   is explicitly set to something other than *11s*
 
 .. [#avmflash]
-  For instructions on how to flash AVM devices, visit https://www.fritzfla.sh
+  For instructions on how to flash AVM devices, visit https://fritz-tools.readthedocs.io/de/latest/
```

Comment:
i think it is intended to keep the easiert url, let's wait for @blocktrron

Reply:
Yeah, I think it's a vanity url that is primarily easy to remember. Let's keep it that way, maybe drop the ẁww.` though :)

Reply:
I am not sure about it, from my point of view i will not remember the short url, when using the readthedocs page, nothing is saying anything about something like the short link (beside of fritz), i will just remember "fritz tools" i guess.

Also short urls might become invalid in the long run, might be used for tracking people, .. i don't know.

Maybe replacing it with https://fritz-tools.readthedocs.io/en/latest/ would be better as the Gluon docs are using en, too?

But i could remove/update the change (as requested) of cause.

Reply:
let's wait for the tool's author so he can decide.

Reply:
yes please

Reply:
I don't consider readthedocs as the final solution, as the performance is rather bad. Also, in case Readthedocs ceases to exist or we have other problems with their service, we can easily switch to a self-hosted solution.

So i would leave the fritzfla.sh vanity URL in place as people are referring to the flash-tool as fritzflash already.

Regarding the domain/hosting situation: This is currently handled by me, but no access logs or anything else are created for it.

Reply:
undone the change but remove `www` as suggested.